spec/support/corpus.rb

```diff
@@ -242,13 +306,24 @@ def system(arguments)
                   s(:guard, s(:or, s(:primitive, TrueClass), s(:primitive, FalseClass)))),
                 s(:key_symbolize, :mutation_generation,
                   s(:guard, s(:or, s(:primitive, TrueClass), s(:primitive, FalseClass)))),
-                s(:key_symbolize, :exclude,             s(:map, s(:guard, s(:primitive, String))))
+                s(:key_symbolize, :expected_errors,
+                  s(:block,
+                    s(:guard, s(:primitive, Hash)),
+                    s(:custom,
+                      [
+                        ->(hash) { hash.map { |key, value| [Pathname.new(key), value] }.to_h  },
+                        ->(hash) { hash.map { |key, value| [key.to_s,          value] }.to_h  }
```

Comment:
Nice catch. I want to PR something to rubocop eventually that flags `ExtraSpacing` for multiple lines which are aligned